.github/pull_request_template.md

```diff
@@ -1,27 +1,18 @@
 ## Description
 
-<!-- Please include a summary of the change. Any relevant motivation or context is also helpful, as well as any dependencies that are required for this change -->
+...
 
 ## Related Issue
 
-<!--- This project prefers to accept pull requests related to open issues -->
-<!--- If suggesting a new feature or change, please discuss it in an issue first -->
-<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
-
-Fixes # (issue)
+...
```

Comment:
I like the suggestion to put a "fixes issue" here and don't think we should drop that.

Reply:
People will often skim the contributor guidelines but the PR template is something that will be shoved in their face and I'd rather encourage best practices like this more directly.